src/schema/filterHelper.js

```diff
@@ -0,0 +1,41 @@
+import { getObjectsFromUrls } from './apiHelper';
+
+export async function filterHelper(connection) {
+  const { args } = connection;
+  if (args.name) {
+    return getObjectsFilteredByName(connection);
+  }
+  if (args.episodeID) {
+    return getObjectsFilteredByID(connection);
+  }
+  if (args.withCharacter) {
+    const people = getObjectsFilteredByName(connection);
+    return getObjectsFromUrls(people[0].films);
+  }
```

Comment:
Currently only one of the filters can be applied, right? Does it make sense to add the possibility to apply multiple filters in the same request?